### DISCUSSION TITLE
Is there a way to use Pluto with vs-code notebooks?

### DISCUSSION THREAD

**Discussion by dylanxyz** [fonsp/Pluto.jl#2084] [Questions about using Pluto] [answered]
Visual Studio Code has now support for [Jupyter notebooks](https://code.visualstudio.com/docs/datascience/overview), which is very convenient for people who use vs-code all day. I really enjoy using Pluto, but i'm afraid it lacks the editing abilities that vs-code (and its extensions) provide when writing code. Is there any way to use the vs-code notebook front-end with Pluto as a backend?

**Comment by fonsp** [comment, marked as answer]
Not yet possible! But I do think that it would be possible to make this integration, also with `@bind` support. Get in touch if you are interested!
